libr/core/canal.c

```diff
@@ -2525,6 +2525,33 @@ static int fcn_print_legacy(RCore *core, RAnalFunction *fcn) {
 	RList *refs, *xrefs;
 	int ebbs = 0;
 	char *name = r_core_anal_fcn_name (core, fcn);
+
+	// execution times:
+	r_cons_printf ("execution state: ");
+	if (core->dbg->trace->enabled == 0) {
+		r_cons_printf ("(dbg.trace unset)\n");
+	} else if (core->dbg->trace->enabled == 1) {
```

Comment:
this is boolean and can only contain 2 values,  no need to == 0  and == 1, just if{}else{}